### DISCUSSION TITLE
Auto Redeem

### DISCUSSION THREAD

**Discussion by fr3shpr1nc4** [Vedza/NitroSniperGo#133] [Q&A] [answered]
this might be a stupid question, but does this sniper auto redeem nitro for the main account or do I have to redeem it manually?

**Comment by Vedza** [comment, marked as answer]
Hi,
Of course the sniper redeem it automatically.

**Reply by fr3shpr1nc4** [reply]
thanks, much appreciated :)

**Reply by nuggetmonkey** [reply]
It auto redeems it on the main right?

**Reply by fr3shpr1nc4** [reply]
yup! I was able to get a code within the first few hours of sniping on my alt, I now have nitro on my main :)
